[2023-09-11 12:01:17,426] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: listing_4_15.extract_gz scheduled__2023-09-11T08:00:00+00:00 [queued]>
[2023-09-11 12:01:17,439] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: listing_4_15.extract_gz scheduled__2023-09-11T08:00:00+00:00 [queued]>
[2023-09-11 12:01:17,440] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-09-11 12:01:17,440] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2023-09-11 12:01:17,440] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-09-11 12:01:17,455] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): extract_gz> on 2023-09-11 08:00:00+00:00
[2023-09-11 12:01:17,461] {standard_task_runner.py:52} INFO - Started process 1355 to run task
[2023-09-11 12:01:17,466] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'listing_4_15', 'extract_gz', 'scheduled__2023-09-11T08:00:00+00:00', '--job-id', '515', '--raw', '--subdir', 'DAGS_FOLDER/listing_4_15.py', '--cfg-path', '/tmp/tmpzipxjwt1', '--error-file', '/tmp/tmpfyafsde5']
[2023-09-11 12:01:17,466] {standard_task_runner.py:80} INFO - Job 515: Subtask extract_gz
[2023-09-11 12:01:17,537] {task_command.py:371} INFO - Running <TaskInstance: listing_4_15.extract_gz scheduled__2023-09-11T08:00:00+00:00 [running]> on host b18943745c0d
[2023-09-11 12:01:17,622] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=listing_4_15
AIRFLOW_CTX_TASK_ID=extract_gz
AIRFLOW_CTX_EXECUTION_DATE=2023-09-11T08:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-09-11T08:00:00+00:00
[2023-09-11 12:01:17,623] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2023-09-11 12:01:17,624] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'gunzip --force /tmp/wikipageviews.gz']
[2023-09-11 12:01:17,635] {subprocess.py:85} INFO - Output:
[2023-09-11 12:01:18,483] {subprocess.py:92} INFO - 
[2023-09-11 12:01:18,484] {subprocess.py:92} INFO - gzip: /tmp/wikipageviews.gz: unexpected end of file
[2023-09-11 12:01:18,503] {subprocess.py:96} INFO - Command exited with return code 1
[2023-09-11 12:01:18,517] {taskinstance.py:1909} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/bash.py", line 195, in execute
    f'Bash command failed. The command returned a non-zero exit code {result.exit_code}.'
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2023-09-11 12:01:18,520] {taskinstance.py:1420} INFO - Marking task as FAILED. dag_id=listing_4_15, task_id=extract_gz, execution_date=20230911T080000, start_date=20230911T120117, end_date=20230911T120118
[2023-09-11 12:01:18,535] {standard_task_runner.py:97} ERROR - Failed to execute job 515 for task extract_gz (Bash command failed. The command returned a non-zero exit code 1.; 1355)
[2023-09-11 12:01:18,565] {local_task_job.py:156} INFO - Task exited with return code 1
[2023-09-11 12:01:18,610] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
